src/test/java/Murat_1/Murat_Test_1.java

```diff
@@ -0,0 +1,12 @@
+package Murat_1;
+
+public class Murat_Test_1 {
+
+    // code 1
+    // code 1
```

Comment:
hi murat, is this code1 used twice? is it necessary? thanks

Reply:
I thik code 2 must be also reviewed

Reply:
Hi, Mehmet bey
yea, this code is necessary! pls check ticket !

Reply:
all clear. agree with you.